### DISCUSSION TITLE
EMA %5 Deviation

### DISCUSSION THREAD

**Discussion by SevketBedir** [DaveSkender/Stock.Indicators#1168] [Help and support]
Hey Guys, 

First thanks a lot for lib, 

is there anyway to decrease %5 deviation ?

**Comment by DaveSkender** [comment]
Are you looking for a way to shift the `Ema` output value by 5%?

**Reply by SevketBedir** [reply]
EMA output value not match values which calculated by tradingview or binance, 
we are trying to get the same values with binance, 

By the way, it is working (Thanks a lot, great library 🙏🏻🙏🏻) and we can get really close result,
we are just want to know, is there anyway for us to improve it ?

**Comment by DaveSkender** [comment]
A solution is to use significantly more historical quotes to allow a longer warmup period (and then don't use those prior results).  Indicators like EMA have a mathematical convergence that has improving accuracy over time.  The reason you don't see these less accurate warmup periods in charting platforms is because they're not showing this _prior data_.  More info:

- #688
- #801
